subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -21,6 +21,24 @@
  ****************************************************************************/
 #define DOUBLE_BUFFER_SIZE 2
 
+#if defined(CONFIG_BT_TICKER_EXT)
+#if !defined(CONFIG_BT_CTLR_ADV_AUX_SET)
+#define BT_CTLR_ADV_AUX_SET 0
+#else
+#define BT_CTLR_ADV_AUX_SET CONFIG_BT_CTLR_ADV_AUX_SET
+#endif
+#if !defined(CONFIG_BT_CTLR_ADV_SYNC_SET)
+#define BT_CTLR_ADV_SYNC_SET 0
+#else
+#define BT_CTLR_ADV_SYNC_SET CONFIG_BT_CTLR_ADV_SYNC_SET
+#endif
+#if defined(CONFIG_BT_CTLR_ADV_ISO)
+#define TICKER_EXPIRE_INFO_MAX (BT_CTLR_ADV_AUX_SET + BT_CTLR_ADV_SYNC_SET*2)
+#else
+#define TICKER_EXPIRE_INFO_MAX (BT_CTLR_ADV_AUX_SET + BT_CTLR_ADV_SYNC_SET)
```

Comment:
This means that if a new use case comes up, e.g. for CIS Central connection spacing, this would need to be changed. I think we need to add some kind of Kconfig for additional expire info entries, or skip the calculation and have a value set in Kconfig, with a value described in a comment.
We can do this later - not blocking for approval.

Reply:
I will prefer allocations for the extended features in ticker to come from caller modules. But for now, I did not have objections the way it  is now. Yes, definitely we should try move use of Controller Kconfigs inside ticker and may be have ticker specific Kconfig that can have the values inherited.